src/wam2layers/tracking/backtrack.py

```diff
@@ -91,16 +97,10 @@ def backtrack(
     # At this point any of the storages could still be overfull, thus stabilize and assigns losses
```

Comment:
At this point the storages could still be overfull _internally_, thus stabilize and assign _internal_ losses